sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/LinkedEntityImpl.java

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.implementation.models;
+
+import com.azure.ai.textanalytics.models.LinkedEntity;
+import com.azure.ai.textanalytics.models.LinkedEntityMatch;
+import com.azure.core.annotation.Immutable;
+import com.azure.core.util.IterableStream;
+
+/**
+ * The {@link LinkedEntityImpl} model.
+ */
+@Immutable
+public final class LinkedEntityImpl implements LinkedEntity {
+    /*
+     * Entity Linking formal name.
+     */
+    private final String name;
+
+    /*
+     * List of instances this entity appears in the text.
+     */
+    private final IterableStream<LinkedEntityMatch> linkedEntityMatches;
+
+    /*
+     * Language used in the data source.
+     */
+    private final String language;
+
+    /*
+     * Unique identifier of the recognized entity from the data source.
+     */
+    private final String dataSourceEntityId;
+
+    /*
+     * URL for the entity's page from the data source.
+     */
+    private final String url;
+
+    /*
+     * Data source used to extract entity linking, such as Wiki/Bing etc.
+     */
+    private final String dataSource;
+
+    /**
+     * Creates a {@link LinkedEntityImpl} model that describes linked entity.
+     *
+     * @param name The entity Linking formal name.
+     * @param linkedEntityMatches A list of instances this entity appears in the text.
+     * @param language The language used in the data source.
+     * @param dataSourceEntityId Unique identifier of the recognized entity from the data source.
+     * @param url URL for the entity's page from the data source.
+     * @param dataSource The data source used to extract entity linking, such as Wiki/Bing etc.
+     */
+    public LinkedEntityImpl(String name, IterableStream<LinkedEntityMatch> linkedEntityMatches, String language,
+                            String dataSourceEntityId, String url, String dataSource) {
+        this.name = name;
+        this.linkedEntityMatches = linkedEntityMatches;
+        this.language = language;
+        this.dataSourceEntityId = dataSourceEntityId;
+        this.url = url;
+        this.dataSource = dataSource;
+    }
+
+    /**
+     * Get the name property: Entity Linking formal name.
+     *
+     * @return The name value.
+     */
+    public String getName() {
+        return this.name;
+    }
+
+    /**
+     * Get the linkedEntityMatches property: List of instances this entity appears in the text.
+     *
+     * @return The linkedEntityMatches value.
+     */
+    public IterableStream<LinkedEntityMatch> getLinkedEntityMatches() {
+        return this.linkedEntityMatches;
```

Comment:
in python we call this property `matches`, just want to make sure it's ok that Java diverges

Reply:
good catch. I will update it.